client/js/libs/canvas/middleware.js

```diff
@@ -53,8 +53,13 @@ function proxyCanvas(store, action, params) {
       original : action,
       lastPage,
       response,
-      error
     }); // Dispatch the new data
+  }).catch((error) => {
+    store.dispatch({
+      type     : action.canvas.type + DONE,
```

Comment:
Even Justin says not to do this. Align it if you must, but don't use the colon to align it. Align with spaces only please.